MovieStoreWebApi/Applications/MovieOperations/Commands/CreateMovie/CreateMovieCommand.cs

```diff
@@ -7,8 +7,11 @@ namespace MovieStoreWebApi.Applications.MovieOperations.Commands.CreateMovie
     public class CreateMovieCommand
     {
         public CreateMovieModel Model { get; set; }
+        public const string ExceptionMessage = "Film zaten mevcut.";
```

Comment:
const field olduğu için bunu classın en üst satırına alalım, Model field'ıyla da boşluk olsun arasında.

Reply:
Düzeltildi.